tests/test_web_log.py

```diff
@@ -45,8 +45,8 @@ def test_access_logger_format() -> None:
 )
 def test_access_logger_time(monkeypatch) -> None:
     now = datetime.datetime(1843, 1, 1, 0, 30)
-    mock_datetime = mocker.patch("aiohttp.datetime.datetime")
-    mock_timezone = mocker.patch("aiohttp.timezone")
+    mock_datetime = monkeypatch.patch("aiohttp.datetime.datetime")
```

Comment:
I believe you should use `setattr()`: https://docs.pytest.org/en/latest/monkeypatch.html